x-team/xp-ui
# Cleanup ApplicantBadge and ApplicantGrid

**Release Type:** *Dev Improvements* 

Fixes https://x-team-internal.atlassian.net/browse/XP-2879

## Description

- Remove all relations to tabular mode at ApplicantBadge and ApplicantGrid.
- Add Flow notation to ApplicantBadge (fixes https://github.com/x-team/xp/issues/1853 cleanup).

## Checklist


## Steps to Test or Reproduce

1. run `npm start`
1. Try out `ApplicantBadge` stories
1. Try out `ApplicantGrid` stories
1. Also try the xp-ui lib on app and observe everything working as expected

## Impacted Areas in Application

- `src/components/forms/ApplicantListFilter.js`
- `src/components/forms/ApplicantListFilter.stories.js`
- `src/components/ui/ApplicantBadge.js`
- `src/components/ui/ApplicantBadge.stories.js`
- `src/components/ui/ApplicantGrid.js`
- `src/components/ui/ApplicantGrid.stories.js`
- `src/components/ui/ApplicantGridHeader.js`
- `src/components/ui/ApplicantGridHeader.stories.js`
- `src/components/ui/Dropdown.js`
- `src/components/ui/ListExclusionFormPopup.stories.js`
- `src/components/ui/ListsScreen.js`
- `src/components/ui/ListsScreen.stories.js`
- `src/index.js`
